coalib/coala_modes.py

```diff
@@ -1,3 +1,6 @@
+import logging
+
+
 def mode_normal(console_printer, log_printer, args, debug=False):
```

Comment:
@arpit7714 , Logging is not used anywhere else in the file and you see in the file each functions have their own import so, it would be better if you import logging here :)